docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,158 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+*coala* has an awesome testing framework so you can write tests for bears
+with ease.
+
+You can use the following to test your bears:
+
+- ``LocalBearTestHelper.check_validity``
+- ``LocalBearTestHelper.check_results``
+- ``verify_local_bears``
+
+Understanding through examples
+------------------------------
+
+Let us understand how to write tests for ``MyBear`` written for
+``python``.
+
+.. code::
+
+    from coalib.results.Result import Result
+    from coalib.bears.LocalBear import LocalBear
+
+
+    class MyBear(LocalBear):
+
+        def run(file,filename):
+            if(len(file)>10):
+                yield Result(self, "Too many lines")
+
+**EXAMPLE 1** using ``verify_local_bear``
+
+.. code::
+
+    from bears.python.MyBear import MyBear
+    from tests.LocalBearTestHelper import verify_local_bear
+
+
+    good_file = '''
```

Comment:
IMO you could just do 
```
good_file = '1\n2\n3\n'.splitlines
bad_file = '1\n2\n3\n4\n5\n6\n7\n8\n9\n10\n11\n'.splitlines()
```
makes it readable and clear that the here the bad file is bad since it has 11 lines